docs/atomic/particle_class.rst

```diff
@@ -30,7 +30,7 @@ but element names and many aliases are not.
 >>> hydride = Particle('H-')
 
 An `int` may be used as the first positional argument to
-`plasmapy.atomic.Particle` to represent an atomic number.  For isotopes
+`~plasmapy.atomic.Particle` to represent an atomic number.  For isotopes
```

Comment:
Just a question. What does the `~` do, again? Why do some of these links have the `~` character and some don't need to? Does it signify an external (non-standard lib) import?

Reply:
Without putting in the `~`, the doc build here wrote out the whole "plasmapy.atomic.Particle".  With the `~`, it just writes out `Particle`.  I think it lets ReST or Sphinx know that it's a link to a particular object.  I'm by no means a ReST expert though!